R/run_eggla_gwas.R

```diff
@@ -508,6 +518,42 @@ run_eggla_gwas <- function(
         "--out", results_file
       ), collapse = " "))
 
+      if (!quiet) message(sprintf("[%s] Computing PLINK2 frequencies ...", basename(vcf)))
+      system(paste(c(
+        bin_path[["plink2"]],
+        "--vcf", vcf_file, "dosage=DS",
+        "--mach-r2-filter",
+        "--threads", threads,
+        "--freq",
+        if (file.exists(sprintf("%s.samples", basename_file))) c("--keep", sprintf("%s.samples", basename_file)),
+        "--silent",
+        "--out", results_file
+      ), collapse = " "))
+
+      if (!quiet) message(sprintf("[%s] Computing PLINK2 missing rate ...", basename(vcf)))
+      system(paste(c(
+        bin_path[["plink2"]],
+        "--vcf", vcf_file, "dosage=DS",
+        "--mach-r2-filter",
+        "--threads", threads,
+        "--missing",
+        if (file.exists(sprintf("%s.samples", basename_file))) c("--keep", sprintf("%s.samples", basename_file)),
+        "--silent",
+        "--out", results_file
+      ), collapse = " "))
```

Comment:
Missing rate (1 - Call rate) report by PLINK2 using the same input as for GLM.

Reply:
Follow-up on <https://github.com/mcanouil/eggla/pull/97#discussion_r1115505751>.